apps/fishing-map/features/datasets/upload/NewTrackDataset.tsx

```diff
@@ -78,35 +81,29 @@ function NewTrackDataset({
     if (file) {
       updateFileData(file)
     } else if (dataset) {
-      console.log('TODO', dataset)
-      setDatasetMetadata((meta) => ({
-        ...meta,
+      setDatasetMetadata({
         id: dataset.id,
         name: dataset.name,
         public: true,
```

Comment:
I think this should be taken from the dataset name instead of hardcoded, could use the existing `isPrivateDataset` function

Reply:
What should be the value in newly created datasets?

Reply:
It is public by default